Shared/Level/Level.gd

```diff
@@ -18,6 +18,7 @@ func _ready():
 
 	for point in $CapturePoints.get_children():
 		var new_scene = capture_point_scene.instance()
+		new_scene.set_name("I".repeat(point.get_index() + 1))
```

Comment:
Really not the best way of doing it, as 4 capture points would break it. We probably will never have 4 capture points, so I guess it's fine, though